src/task/script.js

```diff
@@ -1,4 +1,6 @@
 const API_BASE_URL = window.API_BASE_URL;
+const assigneeAvatar = document.getElementById('assigneeAvatar');
```

Comment:
These should also move to constant. So, all the hardcode values should be part of the Constant.

Reply:
moved API_BASE_URL to constants, assigneeAvatar can also be moved to constants, but as you suggested for different types of constants we should have different constants files, and now we are working on the restructuring of whole code, so once that is done, will move the HTML element also to constant